app/assets/javascripts/notebook/magic/pivotChart.coffee

```diff
@@ -20,9 +20,38 @@ define([
                                     eval("var _f_ = " + val)
                                     _f_
                                 )
+
+      get_cell = () ->
+        # get cell object, like done in get_cell_elements in notebook.js
+        cell_dom_element = $(container).parents(".cell").not('.cell .cell')[0]
+        $(cell_dom_element).data("cell")
+
+      get_saved_pivot_state = () ->
+        saved_state = get_cell().metadata.presentation?.pivot_chart_state
+        JSON.parse(saved_state || "{}") || {}
```

Comment:
if for some weird reason saved state was non empty & not JSON parseable, this raise exception